fastlane/Fastfile

```diff
@@ -128,6 +128,7 @@ platform :ios do
       number_of_retries: 5,
       result_bundle: true,
       testplan: "CI-AllTests",
+      configuration: 'Debug',
```

Comment:
⚠️ Explicitly setting these to Debug.

`Fastlane`'s `scan` documents that `Release` is the default. However, if we don't pass the value, `scan` wasn't passing any configuration to `xcodebuild`, so it was relying on the `scheme`'s default, which is `Debug`.
Instead of relying on that, which might change in the future, this makes the configuration explicit.